tests/test_jiralib.py

```diff
@@ -0,0 +1,65 @@
+# This file is part of the github2jira project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2021 Red Hat, Inc.
+#
+
+import os
+
+from unittest import mock
+
+from unittest.mock import patch
+import pytest
+
+from github2jira.githublib import Issue
+import tests.common as common
+from tests.common import get_raw_issue
+
+
+def mockenv(**envvars):
+    return mock.patch.dict(os.environ, envvars)
+
+
+class MockedJIRA:
+    def __init__(self, options=None, basic_auth=None):
+        return
+
+
+@pytest.fixture
+def mocked_jira():
+    return MockedJIRA()
+
+
+@mockenv(
+    JIRA_SERVER="dummy",
+    JIRA_USERNAME="dummy",
+    JIRA_TOKEN="dummy",
+    JIRA_PROJECT="dummy",
+    JIRA_PROJECT_ID=common.PROJECT_ID,
+    JIRA_COMPONENT=common.COMPONENT,
+)
+@patch("jira.JIRA")
+def test_initialization(self, mocked_jira):
+    from github2jira.jiralib import Jira, config
```

Comment:
It is a bad practice to import inside functions, please avoid it.

Reply:
yep dirty leftover